authors/apps/authentication/models.py

```diff
@@ -126,5 +128,5 @@ def token(self):
             'email': self.email,
             'exp': int(date.strftime('%s')),
         }
-        token = jwt.encode(payload, settings.SECRET_KEY, algorithm='HS256')
-        return token.decode()
+        token = jwt.encode(payload, settings.SECRET_KEY, algorithm='HS256').decode('utf-8')
```

Comment:
Probably no need to change this anymore

Reply:
I had to put in that part for mt tests to pass